### PR TITLE
Doc string updates

### PR DESCRIPTION
**Pull request by Randy424**
regarding: https://github.com/stolostron/backlog/issues/24824

### CODE REVIEW

File: frontend/public/locales/en/translation.json [hunk 1]
```diff
@@ -300,7 +300,7 @@
     "cim.infra.manage.link": "Manage infrastructure environments",
     "cim.infra.missing.warning.text": "An infrastructure environment with available hosts is needed to proceed with the Discovered Hosts option.",
     "cim.infra.missing.warning.title": "No infrastructure environments available",
-    "cim.onpremise.banner.body": "<p>The best solution for creating cluster on an <bold>On Premise</bold> at scale. Easily create ready to go clusters for your applications.</p><p>Easily create and re-create clusters from hosts that are provided by an infrastructure environment.</p>",
+    "cim.onpremise.banner.body": "<p>The best solution for creating cluster on an <bold>on-premise</bold> at scale. Easily create ready to go clusters for your applications.</p><p>Easily create and re-create clusters from hosts that are provided by an infrastructure environment.</p>",
```
Comment (Randy424) [inline frontend/public/locales/en/translation.json:303]: The "on-premise" change.
FYI @swopebe

**Comment by Randy424** [comment]
Here are our notes from the meeting: https://docs.google.com/document/d/1MNapPaEefm06O33EcWC68syw2pHYmfTaQ_6TwlsHR6Q/edit?usp=sharing

File: frontend/public/locales/en/translation.json [hunk 2]
```diff
@@ -168,7 +168,7 @@
     "bareMetalAsset.emptyState.subtitle": "Click <bold>Create bare metal asset</bold> or <bold>Import bare metal assets</bold> to create your resource.",
     "bareMetalAsset.emptyState.title": "You don't have any bare metal assets",
     "bareMetalAsset.importAction.button": "Open file",
-    "bareMetalAsset.importAction.message": "The first line of the CSV file must be a comma separated header row that defines the following columns: hostName, hostNamespace, bmcAddress, macAddress, role (optional), username, password.",
+    "bareMetalAsset.importAction.message": "The first line of the CSV file must be a comma separated header row that defines the following columns: hostName, hostNamespace, bmcAddress, macAddress, role Optional:, username, password.",
```
Comment (swopebe) [inline frontend/public/locales/en/translation.json:171]: "The first line of the CSV file must be a comma separated header row that defines the following columns: hostName, hostNamespace, bmcAddress, macAddress, role. Optional: you can define username, password." 

(Note the change to two complete sentences, some way to get Optional: out of the middle?)

Could it be? "Optional columns to define: username, password."

**Review by swopebe — APPROVED** [review]

File: frontend/public/locales/en/translation.json (hunk 2 above)
Comment (swopebe) [reply frontend/public/locales/en/translation.json:171]: @Randy424 one comment above in that confusing sentence, otherwise I approved.

Comment (Randy424) [reply frontend/public/locales/en/translation.json:171]: @swopebe 
I think "(optional)" here, was referring to "role". Can we keep it as **_"...hostName, hostNamespace, bmcAddress, macAddress, role (optional), username, password."_**?

Comment (KevinFCormier) [reply frontend/public/locales/en/translation.json:171]: Why was this merged like this without resolving?

**Pull request merged**